docs/lago-tutorial/scenarios/jenkins/README.rst

```diff
@@ -0,0 +1,32 @@
+About
```

Comment:
Also, start the readme with a 'scenario title', like:

```
Jenkins master and slaves scenario
==============================
```

That will add a superlevel to the toctree, and know what the readme is about without having to guess by it's path (as in navigating web pages)